data/plugins/skill/mining/ores.rb

```diff
@@ -0,0 +1,46 @@
+REWARD_ORES = {}
+
+class Ores
```

Comment:
Done* i also changed all the names to make it fit since i just realized it was all over and the names were getting confusing imo